widgets/url-sync/index.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+
+var AlgoliaSearchHelper = require('algoliasearch-helper').AlgoliaSearchHelper;
+
+var isEqual = require('lodash/lang/isEqual');
+var merge = require('lodash/object/merge');
+var forEach = require('lodash/collection/forEach');
+
+var getNow = Date.now ? function() { return Date.now(); } : function() { return (new Date()).getTime(); };
```

Comment:
You can safely use Date.now() everywhere, will be polyfill`ed: https://cdn.polyfill.io/v1/docs/features/#Date_now